### ISSUE TITLE
resolvePluginClass should look for lowercase plugins

### ISSUE THREAD

**Issue by TimMun** [deepstreamIO/deepstream.io#1002]
When loading plugins, the deepstream server code looks for a package names with capital letters in them.

For example, when using the deepstream.io-clusternode-redis plugin, on some OS's, deepstream fails to find the module because it looks for "@deepstream/clusterNode-redis" (note the capital 'N'.) (see @deepstream/server/src/config/config-initialiser.ts). While it may work fine on my local Mac, this won't work on many other systems where the requires are case-sensitive. 

In my case I get the error below when trying to deploy on my kube cluster:

```
Error: Cannot load module @deepstream/clusterNode-redis or deepstream.io-clusterNode-redis
at resolvePluginClass (/app/node_modules/@deepstream/server/src/config/config-initialiser.ts:270)
```

**Comment by yasserf** [comment]
Thanks for raising! This has already been fixed in the V5 branch, which I might port back depending when I do the next release

**Comment by yasserf** [comment]
Should be fixed in v4.2.3
